src/System.Tuples/src/project.json

```diff
@@ -0,0 +1,18 @@
+﻿{
+  "dependencies": {
+    "Microsoft.NETCore.Platforms": "1.0.1-rc3-24011-00",
```

Comment:
FYI, the CI is failing due to verification that all of the project.jsons use the desired version.  The version gets updated daily, and this will need to be kept in sync.